apache/wsgi.conf.in

```diff
@@ -108,3 +108,14 @@ WSGIScriptAliasMatch ^${apache_entry_path}/ ${buildout:directory/buildout/parts/
 
 # kml file downloads
 AliasMatch ^${apache_entry_path}/kml/(map\.geo\.admin\.ch_KML_\d+\.kml)$ ${print_temp_dir}/kml/$1
+
+# static downloads from zadara
+<Directory "/var/local/cartoweb/downloads/>
+    Options Indexes FollowSymLinks
```

Comment:
I think we should not allow Indexes, as this allows anybody to browser the complete zadara drive under /var/local/cartoweb/downloads/

`Options -Indexes FollowSymLinks`